src/test/java/seedu/address/logic/handler/DuplicatePhoneTaggerTest.java

```diff
@@ -114,6 +112,15 @@ public void execute_tagPhoneDuplicates() {
                         .anyMatch(tag -> tag
                                 .tagName
                                 .equals(DuplicatePhoneTagger.DUPLICATE_PHONE_TAG_NAME))));
```

Comment:
While this works (and I assume this is a proper use of streams), are there any alternative ways of writing this `assertTrue` test such that it won't be so deeply nested?

Reply:
A very good suggestion, I shall ponder it over the following days.